This patch is needed to avoid double definitions of functions
especially when building with security flags turned on. The double
definitions causes the sed.sh script in configure to fail since it
starts to spit out double outputs e.g.

wi_cv_gethostname_size_t size_t size_t

which then caused almost all subsequent compile time tests to fail since
this gets into confdefs.h file

removing this include causes only one definitions to be emitted into
the genrated protos.h file and thus avoiding the above failure.

Other solution would to fix sed.sh to ignore double definitions

Upstream-Status: Pending

Signed-of-by: Khem Raj <raj.khem@gmail.com>
--- a/autoconf_local/aclocal.m4
+++ b/autoconf_local/aclocal.m4
@@ -4220,7 +4220,6 @@ changequote({{, }})dnl
 cat << 'EOF' > "$wi_tmpdir/unistd.c"
 #include <confdefs.h>
 
-#include <unistd.h>
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <netinet/in.h>
